build.cmd

```diff
@@ -389,7 +389,8 @@ if exist "%ProgramFiles(x86)%\Microsoft Visual Studio 12.0\common7\ide\devenv.ex
 if exist "%ProgramFiles%\Microsoft Visual Studio 12.0\common7\ide\devenv.exe" set VisualStudioVersion=12.0
 
 :vsversionset
-if '%VisualStudioVersion%' == '' echo Error: Could not find an installation of Visual Studio && goto :failure
+if "%VisualStudioVersion%" == "" echo Error: Could not find an installation of Visual Studio && goto :failure
+if "%VSSDK150INSTALL%" == "" (if "%BUILD_VS%" == "1" echo Error: Could not find an installation of Visual Studio 2015 VSSDK which is required for building VS components && goto :failure)
```

Comment:
I thought with the move to VSSDK nuget, we shouldnet need this whole version check? we would never use the local VSSDK?